File: Main4/1013.js
var input = require('fs').readFileSync('stdin', 'utf8');

var [A, B, C] = input.split(" ");

var mab = (parseInt(A)+parseInt(B)+Math.abs(parseInt(A)-parseInt(B)))/2;
var mbc = (mab+parseInt(C)+Math.abs(mab-parseInt(C)))/2;

console.log((mbc)+" eh o maior");

File: C/1013.c
#include <stdio.h>
#include <stdlib.h>

int main() {

    int a, b, c;
    scanf("%d", &a);
    scanf("%d", &b);
    scanf("%d", &c);

    int mab = (a + b + abs(a - b)) / 2;
    int mbc = (mab + c + abs(mab - c)) / 2;

    printf("%d eh o maior\n", mbc);

    return 0;
}